anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -498,35 +576,32 @@ def sync(
                     group_download_result,
                     full_flush=full_flush,
                     local_repo=fetched_data,
-                    operation_id=operation_id,
                 )  # Each group sync is a transaction
-                result["groups"].append(new_data)
-            except Exception as e:
+                result.groups.append(new_data)
+            except Exception:
```

Comment:
Is there anything in the exception itself that you'd want to log here?

Reply:
The exception is actually logged by some magic in logger.exception in the exception handler in _sync_group

Reply:
The exception is re-raised there so that the failure count can be updated here by the caller